force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html

```diff
@@ -22,12 +22,12 @@ <h3 slot="title">
         </lightning-layout-item>
       </lightning-layout>
     </div>
-    <lightning-datatable
+    <c-poc-lightning-datatable
```

Comment:
Likely to break the build without `c-poc-lightning-datatable` in the codebase. You're on the right track though!

Reply:
Good catch! I'll fix.

Reply:
Fixed in 223b066.